counterfab/src/main/java/com/andremion/counterfab/CounterFab.java

```diff
@@ -83,6 +83,12 @@ public Float get(CounterFab object) {
     private float mTextHeight;
     private ObjectAnimator mAnimator;
 
+    private int badgePosition = 0;
```

Comment:
Instead of set to 0 you could set it to `RIGHT_TOP_POSITION`

Reply:
done